05/ht/eventBus/eventBus.test.js

```diff
@@ -0,0 +1,78 @@
+const EventBus = require("./script");
+
+const eventBus = new EventBus();
+
+describe("EventBus tests", () => {
+  test("EventBus is function", () => {
+    expect(typeof EventBus).toBe("function");
+  });
+  test("listEvent is object", () => {
+    expect(typeof eventBus.listEvent).toBe("object");
+  });
+  test("method '.on' is 2 arguments", () => {
+    expect(eventBus.on.length).toBe(2);
+  });
+  test("method '.off' is 2 arguments", () => {
+    expect(eventBus.off.length).toBe(2);
+  });
+  test("method '.trigger' is 1 argument", () => {
+    expect(eventBus.trigger.length).toBe(1);
+  });
+  test("triger transfers 2 param in method 'on'", () => {
+    let a = 0;
+    eventBus.on("click", (x, y) => (a = x + y));
+    eventBus.trigger("click", 3, 4);
+    expect(a).toBe(7);
+  });
```

Comment:
Нужно использовать `jest.fn()` функции в таких ситуациях https://jestjs.io/ru/docs/mock-functions
```suggestion
  test("triger transfers 2 param in method 'on'", () => {
    const fn1 = jest.fn()
    const fn2 = jest.fn()
    eventBus.on("click", fn1);
    const args = [1,2,3]
    eventBus.trigger("click", ...args);
    expect(fn1).toHaveBeenCalledTimes(1);
    expect(fn2).toHaveBeenCalledTimes(0);
    expect(fn1).toHaveBeenCalledWith(args);
  });
```

Передейлай ниже на подобный формат. Если чего-то не хватает смотри доки https://jestjs.io/ru/docs/expect